service/src/main/java/bio/terra/workspace/app/controller/ReferencedGcpResourceController.java

```diff
@@ -598,6 +598,7 @@ public ResponseEntity<ApiCloneReferencedGcpGcsObjectResourceResult> cloneGcpGcsO
             .cloneReferencedResource(
                 sourceReferencedResource,
                 body.getDestinationWorkspaceId(),
+                UUID.randomUUID(), // resourceId is not pre-allocated for individual clone endpoints
```

Comment:
nit: Would it be worth making a variant of this endpoint that fills in the UUID, instead of generating one at every use?